index.js

```diff
@@ -119,13 +121,31 @@ module.exports = {
       allowObject: true,
     }],
 
+    // This rule is problematic in JSX
```

Comment:
How so?

Reply:
I'm not sure how you comment out JSX. Perhaps you do it by hand so adding spaces is simply part of that process for you.

I use a keyboard shortcut in my IDE to comment blocks of code, which means I can comment out a component and move on immediately. I don't have to slow down and manually insert spaces to the start and end of my commented out JSX component. It doesn't make the JSX comment more readable, it just makes it annoying to deal with.

I've tried to configure the rule using exceptions and markers and nothing works.

```json
'spaced-comment': ['warn', 'always', {block: {exceptions: ['<', '/>']}}],
```

It just doesn't work. You end up with this no matter what:
<img width="389" alt="jsx_comment_warning" src="https://user-images.githubusercontent.com/162252/80058606-afbf6000-8564-11ea-8518-4f475f091d49.png">

Reply:
Using VS code this is not an issue, i'm guessing this is something with the IDE you use. I can see how it would be frustrating.